docs/security/access-control/users-roles.md

```diff
@@ -77,11 +76,10 @@ See [Create role](../api/#create-role).
 
 ## Map users to roles
 
-After creating roles, you map users (or backend roles) to them. Intuitively, people often think of this process as giving a user one or more roles, but in the security plugin, the process is reversed; you select a role and then map one or more users to it.
+If you didn't specify a role while creating your user, you can map a role to it afterwards.
```

Comment:
I'd go plural here for clarity. If you didn't specify roles when you created a user, you can map roles to it afterwards.